docs/changes/cosmology/12092.feature.rst

```diff
@@ -0,0 +1,3 @@
+Added units module for defining and collecting cosmological units and
+equivalencies. The units can be added to the main Astropy units namespace using
```

Comment:
As you noted, this is not quite true.

Reply:
I've removed that portion.

For some future date:  with a module-level``__getattr__`` and the units registry it should be possible for ``add_enabled_units`` to also put the units on the top-level namespace in a way that's compatible with ``_UnitsContext`` (context manager returned by ``add_enabled_units``).

Reply:
True, though perhaps too much magic? And perhaps worries for thread safety? I guess the idea of enabling units was mostly that they can be used in strings, and thus in storing them inside files, etc.